frontend/src/components/checkbox/checkBox.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ChangeEventHandler, ForwardedRef, ReactNode } from 'react';
```

Comment:
You're still missing the storybook for this component

Reply:
I have just made it